x-pack/plugins/security_solution/cypress/tasks/api_calls/rules.ts

```diff
@@ -59,30 +63,34 @@ export const createCustomRule = (
       enabled: false,
       exceptions_list: rule.exceptionLists ?? [],
       tags: rule.tags,
-      ...(rule.timeline.id ?? rule.timeline.templateTimelineId
+      ...(timeline?.id ?? timeline?.templateTimelineId
         ? {
-            timeline_id: rule.timeline.id ?? rule.timeline.templateTimelineId,
-            timeline_title: rule.timeline.title,
+            timeline_id: timeline.id ?? timeline.templateTimelineId,
+            timeline_title: timeline.title,
           }
         : {}),
     },
     headers: { 'kbn-xsrf': 'cypress-creds' },
     failOnStatusCode: false,
   });
+};
 
 export const createEventCorrelationRule = (rule: CustomRule, ruleId = 'rule_testing') => {
   if (rule.dataSource.type === 'indexPatterns') {
+    const riskScore = rule.riskScore != null ? parseInt(rule.riskScore, 10) : undefined;
+    const severity = rule.severity != null ? rule.severity.toLowerCase() : undefined;
+
     cy.request({
       method: 'POST',
       url: 'api/detection_engine/rules',
       body: {
         rule_id: ruleId,
-        risk_score: parseInt(rule.riskScore, 10),
+        risk_score: riskScore,
         description: rule.description,
-        interval: `${rule.runsEvery.interval}${rule.runsEvery.type}`,
-        from: `now-${rule.lookBack.interval}${rule.lookBack.type}`,
+        interval: `${rule.runsEvery?.interval}${rule.runsEvery?.type}`,
```

Comment:
This will render it to `undefinedundefined` if `rule.runsEvery` is nullable. This is why I'm skeptical about making most of the rule fields optional in the `CustomRule` interface.